test/jdk/javax/security/auth/callback/PasswordCallback/CheckCleanerBound.java

```diff
@@ -47,6 +47,7 @@ public static void main(String[] args) throws Exception {
         // Wait to trigger the cleanup.
         for (int i = 0; i < 10 && weakHashMap.size() != 0; i++) {
             System.gc();
+            Thread.sleep(100);
```

Comment:
Looks ok to me although @RogerRiggs commented in the previous PR that 10ms should be sufficient.

Reply:
Alternatively, the loop count could be raised by 10x.
That would keep the typical running time low and still allow for a worst case.
Your choice.